docs/_layouts/global.html

```diff
@@ -184,7 +184,8 @@ <h1 class="title">{{ page.title }}</h1>
                     });
                 };
                 script.src = ('https:' == document.location.protocol ? 'https://' : 'http://') +
-                    'cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML';
+                    'cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js' +
```

Comment:
No,  according to the suggestion of the migration guide(https://www.mathjax.org/cdn-shutting-down/): 
> Note If you have been using https://cdn.mathjax.org/mathjax/latest/, we suggest to switch to a fixed version going forward as this represents best practices to avoid unexpected changes. That is, you will have to change the address manually to a higher number when new versions become available.

Reply:
I had the same concern..But I think we can use the current version unless there is important change in MathJax.